spec/index.html

```diff
@@ -868,6 +868,18 @@ <h3>Converting an <a>Alias Node</a></h3>
 
     <section id="convert-scalar">
       <h3>Converting a <a>YAML scalar</a></h3>
+      
+      <ul>
+        <li>A `null` scalar is kept as-is;</li>
+        <li>An `integer` scalar is kept as-is;</li>
+        <li>
+        A string scalar is surrounded by double quotes, unless
+            <ul>
+              <li>it has been surrounded by double quotes already, in which case they are kept as-is,
+              <li>it has been surrounded by singular quotes, in which case they are replaced by double quotes, and any double quotes in the scalar itself are escaped with the backslash `\` character.</li>
```

Comment:
```suggestion
        A string scalar is surrounded by double-quotes (`&#34;`),
        except in the following cases:
            <ul>
              <li>it has already been surrounded by double-quotes,
                  in which case they are kept as-is</li>
              <li>it has already been surrounded by single-quotes (`&#39;`),
                  in which case the surrounding single-quotes are replaced by
                  double-quotes, and any double-quote in the scalar itself is
                  escaped with the backslash character (`&#92;`)</li>
```

Reply:
Actually, this shouldn't be in here, and was likely merged in error. The YAML model describes different syntactic forms of strings, but this describing relies on the underlying internal representation of a string, which is not syntactic.

The entire bulleted section should be removed.

Reply:
Removed in fa87c16a123503dcc7b677f2bf28c8b2d9c52cc0.